### PR TITLE
add Soundcast

### PR DESCRIPTION
**Pull request by andresgottlieb**
https://github.com/andresgottlieb/soundcast

### CODE REVIEW

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome-electron/blob/master/contributing.md

File: readme.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ Some good apps written with Electron.
 - [Shiba](https://github.com/rhysd/Shiba) - Live preview Markdown app with linting.
 - [Yays](https://github.com/Bahlaouane-Hamza/Yays) - YouTube music player in your menubar.
 - [nuTorrent](https://github.com/LeeChSien/nuTorrent) - BitTorrent client.
+- [soundcast](https://github.com/andresgottlieb/soundcast) - Cast OSX audio to Chromecast from the menubar.
```
Comment (sindresorhus) [inline readme.md]: `OSX` => `OS X`

**Comment by sindresorhus** [comment]
App names on OS X are usually capitalized. You should probably `soundcast` → `Soundcast` to fit into with the other apps.

**Comment by andresgottlieb** [comment]
You're right @sindresorhus, just fixed both observations.

Thanks,

**Pull request closed without merging**